Dockerfile

```diff
@@ -64,3 +64,43 @@ COPY --from=secrets-provider-builder /opt/secrets-provider-for-k8s/secrets-provi
 
 ENTRYPOINT [ "/usr/local/bin/secrets-provider"]
 
+# =================== MAIN CONTAINER (REDHAT) ===================
+FROM registry.access.redhat.com/rhel as secrets-provider-for-k8s-redhat
+MAINTAINER CyberArk Software, Ltd.
+
+ARG VERSION
+
+LABEL name="secrets-provider-for-k8s"
+LABEL vendor="CyberArk"
+LABEL version="$VERSION"
+LABEL release="$VERSION"
+LABEL summary="Store secrets in Conjur or DAP and consume them in your Openshift application containers"
+LABEL description="The Secrets Provider for Kubernetes runs as an init container and fetches \
+the Conjur secrets that pods require from DAP or Conjur"
+
+# Add limited user
+RUN groupadd -r secrets-provider \
+             -g 777 && \
+    useradd -c "secrets-provider runner account" \
+            -g secrets-provider \
+            -u 777 \
+            -m \
+            -r \
+            secrets-provider && \
+    # Ensure plugin dir is owned by secrets-provider user
+    mkdir -p /usr/local/lib/secrets-provider /etc/conjur/ssl /run/conjur /licenses && \
+    # Use GID of 0 since that is what OpenShift will want to be able to read things
+    chown secrets-provider:0 /usr/local/lib/secrets-provider \
+                           /etc/conjur/ssl \
+                           /run/conjur && \
+    # We need open group permissions in these directories since OpenShift won't
+    # match our UID when we try to write files to them
+    chmod 770 /etc/conjur/ssl \
+              /run/conjur
+
+COPY LICENSE.md /licenses
+
+USER secrets-provider
+
```

Comment:
missing this very important line: `COPY --from=secrets-provider-builder /opt/secrets-provider-for-k8s/secrets-provider /usr/local/bin/`

Reply:
I also see that in the authn-client we have also `ARG VERSION` - https://github.com/cyberark/conjur-authn-k8s-client/blob/master/Dockerfile#L116

I think that it's not necessary. can you please validate this and remove it from there if it is?

Reply:
VERSION comes as a build-arg from `./bin/build` which is the full tag

```
docker build --tag "secrets-provider-for-k8s-redhat:${FULL_VERSION_TAG}" \
   --build-arg VERSION="$FULL_VERSION_TAG" \
   --target "secrets-provider-for-k8s-redhat" \
   .
```

According to our [RH complaince docs](https://github.com/conjurinc/docs/blob/master/reference/infrastructure/openshift_registry_upload_guide.md), RH needs the following
```
LABEL version="$VERSION"
LABEL release="$VERSION"
```

Reply:
i know that. the question is whether the `ARG VERSION` line is required. If the answer is no then we should remove it also from the authn-client to remove confusion.

Reply:
yes we need otherwise we get a warning during build that a variable was not given a value

Reply:
`[Warning] One or more build-args [VERSION] were not consumed`

Reply:
also when pushed to Openshift, the scan fails on lack of version